data/blog/electron-5-0-timeline.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Electron v5.0.0 Timeline
+author: sofianguy
+date: '2019-01-25'
+---
+For the first time ever, Electron is excited to publicize the 5.0.0 release schedule. This is our first step in having a long projected timeline.
```

Comment:
Nit: You're intending to say this is the first time we've had a public release schedule, but what the words read is that this is the first time we've publicized the 5.0.0 release schedule

Reply:
'a long projected timeline'

This phrase doesn't work. Maybe "This is our first step towards having a public, long-term timeline"?